.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+# For pngquant see: https://pngquant.org
+
+
+repos:
+  - repo: https://github.com/gdubicki/pre-commit-pngquant
+    rev: 9fad42f9df2ec8306c73d3e7c0628893ead0ebeb # Use the sha / tag you want to point at
+    hooks:
+      - id: pngquant
+        files: .*\.(png|PNG|jpg)$ 
```

Comment:
does pngquant also support jpg?

Reply:
To be honest, it wasn't mentioned anywhere explicitly or even denied. Therefore, to give it a try, I included it.

Reply:
where did you find out about this `files` option in the first place? I don't see it being documented at https://github.com/gdubicki/pre-commit-pngquant
From a quick glance at the implementation https://github.com/gdubicki/pre-commit-pngquant/blob/main/pngquant-wrapper.sh I guess only `.png` is supported anyway

Reply:
I remember finding it when searching for rev. But sorry, I cannot find the page now. I guess I will update similar to the already available one.